### ISSUE TITLE
Text input fields and the autocomplete attribute 

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3095]
**HC Assessment Label: Serious**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest
URL: https://forms-staging.cdssandbox.xyz/en/id/134
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
All the text input fields have the "autocomplete" attribute set to "off" in the "<input>"  elements. Could you please confirm if it is intentionally done to prevent browsers from offering to autocomplete or auto-fill the fields based on the saved information previously?
For example, see code snippet below.

<label data-testid="label" class="gc-label required" for="14" id="label-14">8. Telephone number<span data-testid="required" aria-hidden="true"> (required)</span></label> 

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Inspect the HTML source for the text input fields

Expected result: If answer is "no" to the question above, to meet the SC 1.3.5 - Identify Input Purpose from WCAG 2.1, the forms fields that are related to collecting personal information directly from and about the user completing the form should have the “autocomplete” attribute with correct value that is outlined in the following document to identify the field’s purpose programmatically. In this form, it can particulary apply to the text input fields in the "Contact information" section.
https://www.w3.org/TR/WCAG21/#input-purposes

**Comment by srtalbot** [comment]
Fixed, now form builders can set the auto-complete value in the user interface. We've also added specific question components like "email" or "name" that sets it automatically.
